tools/deprecated-config-checker/deleted-config.yaml

```diff
@@ -0,0 +1,33 @@
+# This file should contain a list fo deleted config options.
+#
+# The syntax for this file is the same as the syntax for deprecated-config.yaml.
+
+ingester:
+  max_transfer_retries: "Enable the ingester WAL and rely on new ingesters to replay the WAL."
+
+querier:
+  engine:
+    timeout: "Configure global or per-tenant query_timeout limit."
```

Comment:
```suggestion
    timeout: "Configure global or per-tenant query_timeout limit using limits_config."
```

Reply:
same comment for all other runtime configs.

Reply:
Thank you! I think `query_timeout limit inside limits_config` (note **inside** instead of **using**) is more correct.

Reply:
Changed to `Use global or per-tenant query_timeout configuration from limits_config.`